content/en/account_management/billing/custom_metrics.md

```diff
@@ -5,68 +5,199 @@ aliases:
 
-* [Agent Metrics][6]
-* [Directory][7]
-* [Linux Proc Extras][8]
-* [Nagios][9]
-* [PDH Check][10]
-* [Prometheus][11]
-* [SNMP][12]
-* [Windows Services][13]
-* [WMI][14]
+The number of unique tag value combinations submitted for a DISTRIBUTION metric with this tagging scheme is **four**.
 
-Many other integrations can be configured to collect custom metrics, for example:
+* `host:A`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:400`
+* `host:B`, `endpoint:Y`, `status:200`
+
+The number of custom metrics from a [DISTRIBUTION metric][1] is five times the unique combination of metric name, host, and tag values. This results in `request.Latency` reporting a total of **5*4 = 20 custom metrics**:
+
+##### Customized tagging
+
+You can customize [which tag combination][2] aggregations are created for any DISTRIBUTION metric. Say that you want to keep only the `endpoint` and `status` tags associated with your distribution. This leaves the following three unique tag combinations:
+
+* `endpoint:X`, `status:200`
+* `endpoint:X`, `status:400`
+* `endpoint:Y`, `status:200`
+
+The number of custom metrics from a [DISTRIBUTION metric][1] is five times the unique combination of metric name and tag values. As a result of the tag customization, `request.Latency` reporting a total of **5*3 = 15 custom metrics**
+
+##### Percentile aggregations
+
+Percentile aggregations are unique in the way they're counted because **Datadog stores five custom metrics for each potentially queryable tag value combination** to provide you with globally accurate percentiles: `p50`, `p75`, `p90`, `p95`, and `p99`. Suppose you have [added percentile aggregations][3] for `request.Latency` for the set of tags `endpoint` and `status` with the same tag dependency seen earlier. The number of queryable tag value combination is **eight**:
+
+* `endpoint:X`, `status:200`
+* `endpoint:X`, `status:400`
+* `endpoint:Y`, `status:200`
+* `endpoint:X`
+* `endpoint:Y`
+* `status:200`
+* `status:400`
+* `*`
+
+After enabling percentile aggregations for `request.Latency`, this metric name additionally reports **5*8 = 40 custom metrics**.
+
+**Note**:  Only tag value combinations that actually appear in your data are counted as queryable. Since the combination { `endpoint:Y`, `status:400` } was never submitted in your data, this combination won’t be queryable and won’t count towards your custom metric count.
+
+[1]: /developers/metrics/types/?tab=distribution
+[2]: /graphing/metrics/distributions/#customize-tagging
+[3]: /graphing/metrics/distributions/#aggregations
+{{% /tab %}}
+{{< /tabs >}}
+
+## Tracking custom metrics
+
+Administrative users (those with [Datadog Admin roles][4]) can see the total custom metrics per hour and the top 500 custom metrics for their account on the [usage details page][5]. See the [Usage Details][6] documentation for more information.
```

Comment:
Administrative users (those with [Datadog Admin roles][4]) can see the monthly average number of custom metrics per hour and the top 500 custom metrics for their account on the [usage details page][5]. See the [Usage Details][6] documentation for more information.